reflection/serverreflection.go

```diff
@@ -63,9 +64,27 @@ type GRPCServer interface {
 var _ GRPCServer = (*grpc.Server)(nil)
 
 // Register registers the server reflection service on the given gRPC server.
+// Both the v1 and v1alpha versions are registered.
 func Register(s GRPCServer) {
-	svr := NewServer(ServerOptions{Services: s})
-	v1alphagrpc.RegisterServerReflectionServer(s, svr)
+	svr := NewServerV1(ServerOptions{Services: s})
+	v1alphagrpc.RegisterServerReflectionServer(s, asV1Alpha(svr))
+	v1grpc.RegisterServerReflectionServer(s, svr)
+}
+
+// RegisterV1 registers only the v1 version of the server reflection service
+// on the given gRPC server. Many clients may only support v1alpha so most
+// users should use Register instead.
```

Comment:
"... for now" or "... until clients have been upgraded"?

Reply:
Done